compiler/wasm/package.json

```diff
@@ -24,22 +28,51 @@
     "url": "https://github.com/noir-lang/noir/issues"
   },
   "scripts": {
-    "build": "bash ./build.sh",
-    "test": "yarn test:node && yarn test:browser",
-    "test:node": "env TS_NODE_COMPILER_OPTIONS='{\"module\": \"commonjs\"}' mocha",
+    "build": "WASM_OPT=$(bash -c ./wasm-opt-check.sh) webpack",
+    "test": "yarn test:build_fixtures && yarn test:node && yarn test:browser",
+    "test:build_fixtures": "./build-fixtures.sh",
     "test:browser": "web-test-runner",
-    "clean": "chmod u+w web nodejs || true && rm -rf ./nodejs ./web ./target ./result",
+    "test:node": "NODE_NO_WARNINGS=1 mocha --config ./.mocharc.json",
+    "clean": "rm -rf ./build ./target ./dist public/fixtures/simple/target public/fixtures/with-deps/target",
     "nightly:version": "jq --arg new_version \"-$(git rev-parse --short HEAD)$1\" '.version = .version + $new_version' package.json > package-tmp.json && mv package-tmp.json package.json",
     "publish": "echo 📡 publishing `$npm_package_name` && yarn npm publish",
-    "lint": "NODE_NO_WARNINGS=1 eslint . --ext .ts --ignore-path ./.eslintignore  --max-warnings 0",
-    "build:nix": "nix build -L .#noir_wasm",
-    "install:from:nix": "yarn clean && yarn build:nix && cp -rL ./result/noir_wasm/nodejs ./ && cp -rL ./result/noir_wasm/web ./"
+    "lint": "NODE_NO_WARNINGS=1 eslint . --ext .ts --ignore-path ./.eslintignore  --max-warnings 0"
   },
   "devDependencies": {
     "@esm-bundle/chai": "^4.3.4-fix.0",
+    "@ltd/j-toml": "^1.38.0",
+    "@noir-lang/noirc_abi": "workspace:*",
+    "@types/adm-zip": "^0.5.0",
+    "@types/chai": "^4",
+    "@types/mocha": "^10.0.6",
+    "@types/mocha-each": "^2",
+    "@types/node": "^20.10.5",
+    "@types/path-browserify": "^1",
+    "@types/readable-stream": "^4",
+    "@types/sinon": "^17",
+    "@wasm-tool/wasm-pack-plugin": "^1.7.0",
     "@web/dev-server-esbuild": "^0.3.6",
-    "@web/test-runner": "^0.15.3",
-    "@web/test-runner-playwright": "^0.10.0",
-    "mocha": "^10.2.0"
```

Comment:
Can you add this dependency back as we do depend on `mocha`

Reply:
It's just reordering due to `yarn add` adding dependencies alphabetically. It's on line 64